apis/server/v1alpha1/device_types.go

```diff
@@ -166,7 +110,7 @@ type DeviceClass struct {
 
 	// SpecTemplate is a template for the spec of a dynamically provisioned
 	// DeviceSpec.
-	SpecTemplate DeviceParameters `json:"specTemplate"`
+	SpecTemplate DeviceClassSpecTemplate `json:"specTemplate"`
```

Comment:
Embed the `SpecTemplate` so that the `NonPortableClass` interface is satisfied. This will likely be removed due to design changes in https://github.com/crossplaneio/crossplane/pull/926